config/locales/metadata/en.yml

```diff
@@ -432,21 +432,20 @@ en:
 
         data_release_timing:
           label: How is the data release to be timed?
-          help: "Choose from:<p><strong>Immediate:</strong> To be released as soon as possible.</p><p><strong>Standard:</strong> To be released:<ul><li>For managed (EGA) studies: 6 months</li><li>For open (ENA) studies: 12 months</li><li>Transcriptomics studies: on request only</li></ul></p><p><strong>Delayed:</strong><ul><li>For managed (EGA) studies: 6 months plus delay for period</li><li>For open (ENA) studies: 12 months plus delay for period.</li></ul></p><p><strong>Never:</strong> This option is only available if the data release strategy is set to 'Not applicable.'</p>"
+          help: "Choose from:<p><strong>Immediate:</strong> To be released as soon as possible.</p><p><strong>Standard:</strong> To be released:<ul><li>For managed (EGA) studies: 12 months</li><li>For open (ENA) studies: 12 months</li><li>Transcriptomics studies: 12 months</li></ul></p><p><strong>Delayed:</strong><ul><li>For managed (EGA) studies: 12 months plus delay for period</li><li>For open (ENA) studies: 12 months plus delay for period.</li></ul></p><p><strong>Never:</strong> This option is only available if the data release strategy is set to 'Not applicable.'</p>"
 
         data_release_prevention_reason:
           label: What is the reason for preventing data release?
 
         data_release_prevention_approval:
-          label: Has this been approved?
+          label: If reason for exemption requires DAC approval, what is the approval number?
           help: "If this is for data validity reasons: approval from the sponsor is required<br />If this is for legal reasons: approval from the Data Sharing Committee is required (please contact sd4)<br />"
 
         data_release_prevention_reason_comment:
           label: Comment regarding prevention of data release and approval
 
         data_release_delay_reason:
           label: Reason for delaying release
-          help: 'To apply for a delay, please contact <a title="<%= configatron.data_sharing_contact.name %>" href="mailto:<%= configatron.data_sharing_contact.email %>"><%= configatron.data_sharing_contact.email %></a>'
```

Comment:
Can `data_release_delay_approval:` in this file be remove too, for the question which is being removed “Has the delay period been approved by the data sharing committee for this project?”

Reply:
Although it is no longer used for new studies, this metadata is still used on the [study show page](https://github.com/sanger/sequencescape/blob/develop/app/views/shared/metadata/show/_study.html.erb#L52) for existing studies with this value so for backwards compatibility we need to keep it.